test/functional/update/update-no-minversion.bats

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME" 10
+	create_bundle -L -n bundle1 -v 10 -f /file1,/file2,/file3 "$TEST_NAME"
+
+	sudo sed -i "/^minversion:\\t/d" "$WEBDIR"/10/Manifest.MoM
+
+	create_version "$TEST_NAME" 20 10 staging
+
+	update_bundle "$TEST_NAME" bundle1 --add /file4
+	update_bundle "$TEST_NAME" bundle1 --update /file1
+	update_bundle "$TEST_NAME" bundle1 --delete /file2
+}
+
+@test "Update when missing minversion header" {
+
+	run sudo sh -c "$SWUPD update $SWUPD_OPTS"
+	assert_status_is 0
+
+	expected_output=$(cat <<-EOM
+		Update started.
+		Preparing to update from 10 to 20
+		Downloading packs...
+		Extracting bundle1 pack for version 20
+		Statistics for going from version 10 to version 20:
+		    changed bundles   : 1
+		    new bundles       : 0
+		    deleted bundles   : 0
+		    changed files     : 2
+		    deleted files     : 1
+		Starting download of remaining update content. This may take a while...
+		Finishing download of update content...
+		Staging file content
+		Applying update
+		Update was applied.
+		Calling post-update helper scripts.
+		Update successful. System updated from version 10 to version 20
+	EOM
+	)
+	assert_in_output "$expected_output"
```

Comment:
why not _assert_is_output_ instead of _assert_in_output_? the latter will allow more output to be included, if you are not expecting that assert_is_output will make sure only the output you expect is printed.

Reply:
I overlooked this, assert_is_output is leftover from the test that I copied as a starting place... All of your feedback looks good and will be addressed in the next version. Thanks!